src/makeRobot.js

```diff
@@ -37,8 +37,67 @@
  *
  * @return {Robot}
  */
+
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0, y: 0,
+    },
```

Comment:
```suggestion
    coords: {
      x: 0, 
      y: 0,
    },
```
I think it's more readable